src/OpenTK.GLControl/GLControl.cs

```diff
@@ -41,22 +41,24 @@ namespace OpenTK
     /// </summary>
     public partial class GLControl : UserControl
     {
-        private IGraphicsContext context;
-        private IGLControl implementation;
-        private GraphicsMode format;
-        private int major, minor;
-        private GraphicsContextFlags flags;
+        private IGraphicsContext m_context;
+        private IGLControl m_implementation;
+        private readonly GraphicsMode m_format;
+        private readonly int m_major;
+        private readonly int m_minor;
+        private readonly GraphicsContextFlags m_flags;
+        private bool? m_initialVsyncValue;
```

Comment:
Was there a reason all of these were renamed to be prefixed with m_?

Reply:
They should not be named like this in order to conform to our current coding style. Remove the m_ prefix.